src/components/footer/Footer.tsx

```diff
@@ -35,6 +35,24 @@ const Footer = () => {
             Submit Feedback
           </Link>
         </Text>
+        <Text
```

Comment:
This footer is for the older landing page, since we are keeping this for tutorial then its pretty great. We just need to include it in the newer homepage footer

the new homepage footer is at `src/components/home/LandingFooter.tsx`

Reply:
This is just text pointing people to the public url. I don't think we have an obligation to put it on the landing page.